distribution/examples/xslt/README.md

```diff
@@ -5,110 +5,131 @@ With the XSLTInterceptor you can apply XSLT stylesheets to requests and response
 
 #### RUNNING THE EXAMPLE
 
-At the following URL you will get a REST representation of a customer resource.
-
-
-http://www.thomas-bayer.com/samples/sqlrest/CUSTOMER/7/ 
-
-
-In this example we will transform that representation with the XSLTInterceptor. 
-
-
 Execute the following steps:
 
 1. Go to the `examples/xslt` directory.
 
 2. Execute `service-proxy.bat` or `service-proxy.sh`
 
-2. Open the URL http://www.thomas-bayer.com/samples/sqlrest/CUSTOMER/7/ in your browser.
+2. Open the URL https://api.predic8.de/restnames/name.groovy?name=Pia in your browser.
 
-3. Compare it with the response of http://localhost:2000/samples/sqlrest/CUSTOMER/7/
+3. Compare it with the response of http://localhost:2000/restnames/name.groovy?name=Pia
 
 
 ### HOW IT IS DONE
 
-The following part describes the example in detail.  
-
 First take a look at the proxies.xml file.
 
-
-```
+```xml
 <router>
   <serviceProxy port="2000">
     <response>
-      <transform xslt="customer2person.xsl" />
+      <transform xslt="./reformat.xsl" />
     </response>		
-    <target host="www.thomas-bayer.com" port="80" />
+    <target url="https://api.predic8.de/restnames/name.groovy?name=Pia" />
```

Comment:
Target with host an port, so that the user has to call:
localhost/restnames/name.groovy... but he can change the name.